src/scheduler/pbsfs.c

```diff
@@ -188,17 +228,17 @@ main(int argc, char *argv[])
 	else if (flags & FS_DECAY)
 		decay_fairshare_tree(conf.fairshare->root);
 	else if (flags & (FS_GET | FS_SET | FS_ADD | FS_COMP)) {
-		ginfo = find_group_info(argv[2], conf.fairshare->root);
+		ginfo = find_group_info(argv[optind], conf.fairshare->root);
```

Comment:
We haven't validated optind in the case this is FS_ADD. The rest are validated above.

Reply:
It turns out that add has been deprecated for some time.  I'm just going to remove it.  It's not in the 14.2.1 docs.